src/System.IO.FileSystem/src/System/IO/File.cs

```diff
@@ -857,7 +856,7 @@ public static Task WriteAllBytesAsync(string path, byte[] bytes, CancellationTok
 
             using (FileStream fs = new FileStream(path, FileMode.Create, FileAccess.Write, FileShare.Read, DefaultBufferSize, FileOptions.Asynchronous | FileOptions.SequentialScan))
             {
-                await fs.WriteAsync(bytes, 0, bytes.Length, cancellationToken).ConfigureAwait(false);
+                await fs.WriteAsync(new ReadOnlyMemory<byte>(bytes, 0, bytes.Length), cancellationToken).ConfigureAwait(false);
```

Comment:
Can we use the single arg constructor here instead so it avoids unnecessary checsk? Anywhere we have `array, 0, array.Length`, we can simply pass array.

Reply:
Yup, will change.  I did that in most places, but missed this one.